### ISSUE TITLE
CTD on World Congress decision

### ISSUE THREAD

**Issue by valentin-turcu** [LoneGazebo/Community-Patch-DLL#164]
I'm in my 450th turn, Renaissance era, playing as Spain on a Huge map. 
I'm hosting the World Congress and the decision is "Natural Heritage Sites" (to boost natural wonders yields).
After spreading the votes (no matter how I spread them: for or against) the game crashes on the beginning of the next turn (when the election results should become clear and the effects should take place).

Here is my dump file: https://www.dropbox.com/s/t71yammocghm361/CvMiniDump.dmp?dl=0
Here are the logs: https://www.dropbox.com/s/noq7kljn6pnbsjm/Logs.zip?dl=0

**Comment by valentin-turcu** [comment]
Here are the mods I'm running. Only the selected ones in the picture below are activated.

![used_mods](https://cloud.githubusercontent.com/assets/6671776/6743453/bc9a88a0-cea2-11e4-850f-999792a78e32.png)

**Comment by valentin-turcu** [comment]
Here is the autosave right after the CTD (somehow it still managed to autosave) in which clicking the the World Congress resolution notifications always crashes the game.

https://www.dropbox.com/s/cnvmq2p4gdo6jxw/Spain%20Modded%20III%209%20After%20CTD.Civ5Save?dl=0

Please tell me if there's a way I can salvage this save, because I don't think I can start yet another marathon game to abandon mid-save.

**Comment by LoneGazebo** [comment]
Link your lua and database logs please.

**Comment by valentin-turcu** [comment]
Lua log: https://www.dropbox.com/s/rznw90o1zmv8kd9/Lua.log?dl=0
Database log: https://www.dropbox.com/s/78c0b0av1faxf0l/Database.log?dl=0

Hope it helps!

**Comment by LoneGazebo** [comment]
I'll give it a look. 
G

**Comment by LoneGazebo** [comment]
Okay, I've looked at your files, and it isn't good. Your database.log shows a ton of missing promotions from 'something' (not sure what, perhaps an old mod) – also, you don't own Korea, it seems, which is a prerequisite for using the CBP. You might be able to fix  the issue by removing all mods from your MODS folder that you aren't currently using, then clear your cache and reload your save. Other than that, this is most likely an issue of missing files.

**Comment by valentin-turcu** [comment]
I appreciate the effort, thank you very much.

The missing promotions are most likely the result of me trying to use a mod
called "More Promotions" (which, I discovered, had no visible effect, so I
dropped it).

The missing Korea civ is odd however. I'm not at home right now, so I can't
check, but I was pretty sure I owned all the DLC.

I'll try to do what you've said when I get back.

Thank you again.

**Comment by valentin-turcu** [comment]
You were right, I was missing the Korea DLC. 
I've now purchased it, verified the cache integrity to make sure it's downloaded, checked it's active in the DLC list in the main menu and verified I can start a new game with mods as Korea. 

Following that, I cleared all the cache (including mod cache file), unsubscribed from all mods on Steam, and deleted all mods I wasn't using (which means all except IGE and the 2 religion mods).
I've also deleted all the logs to refresh them.

Doing all this, I've tried to load the problematic save and unfortunately, it still crashes. 
Checking the Database.log file, it looks like I've eliminated the missing promotions messages. 

What is **strange**, however, is that despite me now having the Korea DLC for sure and being able to use it, when I try to load my save game the game writes into the Database.log file that 
> [3924.173] Invalid Reference on Civilization_BuildingClassOverrides.CivilizationType - "CIVILIZATION_KOREA" does not exist in Civilizations

It's also saying that 
> [3924.033] Invalid Reference on Building_YieldChanges.BuildingType - "BUILDING_MAUSOLEUM_HALICARNASSUS" does not exist in Buildings 
although I have bought that DLC at the same time as Korea and it's active, I'm 100% sure of it.

Does this mean that the save itself can't recognize any DLC added after it started?
If so, it might actually be impossible to recover the save... unless you have any other idea what I can do.
